### PR TITLE
loglevel in config

### PR DESCRIPTION
**Pull request by giggsoff**
The ability to set loglevel in the config

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: pkg/defaults/defaults.go [hunk 1]
```diff
@@ -92,6 +92,8 @@ const (
 	DefaultInfoRedisPrefix       = "INFO_EVE_"
 	DefaultMetricsRedisPrefix    = "METRICS_EVE_"
 	DefaultRequestsRedisPrefix   = "REQUESTS_EVE_"
+	DefaultEveLogLevel           = "info"
```
Comment (mydatascience) [inline pkg/defaults/defaults.go:95]: Is it the most informative one?

File: pkg/defaults/defaults.go [hunk 2]
```diff
@@ -92,6 +92,8 @@ const (
...
+	DefaultEveRemoteLogLevel     = "warning"
```
Comment (mydatascience) [inline pkg/defaults/defaults.go:96]: Is it the most informative one?

File: pkg/utils/config.go [hunk 3]
```diff
@@ -98,6 +100,9 @@ func InitVars() (*ConfigVars, error) {
 			EServerImageDist:  ResolveAbsPath(viper.GetString("eden.images.dist")),
 			EServerPort:       viper.GetString("eden.eserver.port"),
 			EServerIP:         viper.GetString("eden.eserver.ip"),
+
+			LogLevel:       viper.GetString("eve.loglevel"),
+			RemoteLogLevel: viper.GetString("eve.remoteloglevel"),
```
Comment (mydatascience) [inline pkg/utils/config.go:105]: Why do we differ eve.loglevel and eve.remoteloglevel?

File: pkg/defaults/defaults.go (hunk 1 above)
Comment (giggsoff) [reply pkg/defaults/defaults.go:95]: No, but it is defaults from here: https://github.com/lf-edge/eve/blob/master/docs/CONFIG-PROPERTIES.md

File: pkg/defaults/defaults.go (hunk 2 above)
Comment (giggsoff) [reply pkg/defaults/defaults.go:96]: No, but it is defaults from here: https://github.com/lf-edge/eve/blob/master/docs/CONFIG-PROPERTIES.md

**Pull request merged**